CONRTIBUTING.md

```diff
@@ -61,6 +61,28 @@ __The following approach aims to reconcile these two competing demands:__
 --------
 
 
+### Library Standards
+
+The team has agreed to consistently use several third-party Python packages.
+This list will likely be fairly dynamic as we are always looking for things
+that will increase our productivity. We have a strong preference for tools that
+follow the [Principle of Least Astonishment](http://c2.com/cgi/wiki?PrincipleOfLeastAstonishment)
+(also on [Wikipedia](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)).
+
+The list:
+
+  - [arrow](https://pypi.python.org/pypi/arrow): Date/Time utilities
```

Comment:
Yes, maybe too prescriptive. I would probably support the use of it, but it might not be interesting enough to note in CONTRIBUTING.md. Plus, we have not used it yet in simpl, I don't think, and I would bet we can get by without it.